cipher/ecc.c

```diff
@@ -1471,7 +1499,14 @@ ecc_encrypt_raw (gcry_sexp_t *r_ciph, gcry_sexp_t s_data, gcry_sexp_t keyparms)
   }
 
   if (!rc)
-    rc = sexp_build (r_ciph, NULL, "(enc-val(ecdh(s%m)(e%m)))", mpi_s, mpi_e);
+    {
+      if (DBG_CIPHER)
+        {
+          log_printmpi ("ecc_encrypt res", mpi_s);
+          log_printmpi ("ecc_encrypt public key e", mpi_e);
+        }
+      rc = sexp_build (r_ciph, NULL, "(enc-val(ecdh(s%m)(e%m)))", mpi_s, mpi_e);
+    }
```

Comment:
debuging. can this be dropped?

Reply:
There already are some debug output in the encrypt/decrypt code. New code was added, new debug output was added along with it. What's wrong with that?